src/components/aboutUs/aboutUs.stories.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import AboutUs from "./aboutUs";
+
+storiesOf('AboutUs', module)
+    .add('default', () => {
+        return <AboutUs />
+    });
```

Comment:
Nice job tackling creating a story.  We'll go into that more if we get components that can have more customization.

Reply:
It's pretty awesome! Had to read the storybook doc to understand it more.